docs/kyma/10-08-cant-download-img.md

```diff
@@ -0,0 +1,28 @@
+---
+title: '"Failed to pull image"' error
+type: Troubleshooting
+---
+
+When you try to install Kyma locally on Minikube, the installation can fail at a very early stage logging this error:
+
+  ```
+  POD: tiller-certs-job-cpl67: ERROR: Failed to pull image "eu.gcr.io/kyma-project/test-infra/alpine-kubectl:v20190325-ff66a3a": rpc error: code = Unknown desc = Error response from daemon: Get https://eu.gcr.io/v2/: dial tcp: lookup eu.gcr.io on 192.168.64.1:53: read udp 192.168.64.5:55778->192.168.64.1:53: read: connection refused
+  error
+  POD: tiller-deploy-f88ff7869-z5cmx: ERROR: MountVolume.SetUp failed for volume "tiller-certs" : secrets "tiller-secret" not found
+  ```
+
+This message shows that the installation fails because the required Docker image cannot be downloaded from a Google Container Registry address. Minikube can't download the image because its DNS server cannot resolve the image's address.
```

Comment:
Just thinking - you have `can't` and `cannot` in one sentence - it's is not wrong, but I was just wondering if we should have it unified or it doesn't really matter that much.

Reply:
👍